src/index.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable no-plusplus */
```

Comment:
- Hey, please, let's remove this eslint disable line and try fixing it as this would help improve your development skill. To fix this, update all your iterator with `i++` to `i += 1` and `i--` to `i -= 1` in your code base. I don't want to duplicate this comment all over your codebase.